lazyfs/config/example.toml

```diff
@@ -0,0 +1,37 @@
+[faults]
```

Comment:
I will remove this example since it can indeed mislead into thinking it is expected from multiple faults to be configured. The README explains how to configure these faults.